### ISSUE TITLE
nodejs-4.1.1-1 upgrade fails

### ISSUE THREAD

**Issue by visibilityspots** [TehShrike/k#5]
When upgrading nodejs to version 4.1.1-1 the kanbanize-cli isn't working anymore downgrading to 0.12.7-1 got it working again.

OS is archlinux.

```bash4.1.1-
/usr/lib/node_modules/kanbanize-cli/node_modules/level/node_modules/leveldown/node_modules/bindings/bindings.js:83
        throw e
        ^

Error: Module version mismatch. Expected 46, got 14.
    at Error (native)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at bindings (/usr/lib/node_modules/kanbanize-cli/node_modules/level/node_modules/leveldown/node_modules/bindings/bindings.js:76:44)
    at Object.<anonymous> (/usr/lib/node_modules/kanbanize-cli/node_modules/level/node_modules/leveldown/index.js:1:99)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
```

**Comment by TehShrike** [comment]
Ah, the LevelUP version needs bumped.

**Comment by visibilityspots** [comment]
Is that something we need to do somewhere or?

**Comment by TehShrike** [comment]
Yup, it's right there in the package.json! ;-P

**Comment by TehShrike** [comment]
Published as 1.4.0
